### ISSUE TITLE
issue with the flag NoDestroy

### ISSUE THREAD

**Issue by Railgun69** [Trojaner25/Rocket-Regions#57]
sometimes when a region has the NoDestroy flag on it and there's buildables inside the region i come to find that a lot of buildables lose health randomly, one of my ladders going to my base had 0% health once
every buildable is affected by this bug

**Comment by Trojaner** [comment]
Will investigate

**Comment by jessepowers** [comment]
True.  Buildables take random health hits from anywhere from 30% to 80% without anyone attacking it.  Godmode does the same.  People take random damage and start bleeding out of nowhere when entering/exiting a region.
